### DISCUSSION TITLE
odlr:hasPolicy - Value should be an array

### DISCUSSION THREAD

**Discussion by Wilhelmsson177** [eclipse-edc/Connector#3158] [Ideas]
I tried out the samples and noticed, that the type of the value of the `odlr:hasPolicy` key is changing depending on the amount of policies, when fetching the catalog. When it is only 1 policy it is of type `odlr:Set`, when there are more policies, it becomes an array. Wouldn't it be better to return an array in any way? Even if it is only of length 1?

**Comment by ndr-brt** [comment]
This happens because of the current Json-LD compaction process, it was already discussed in this issue: https://github.com/eclipse-edc/Connector/issues/3137

My advice is, as client maybe is better to expand Json-LD before evaluating it
